### PR TITLE
Update all RSPEC metadata for VB.Net

### PR DESCRIPTION
**Pull request by Evangelink**

### CODE REVIEW

File: sonaranalyzer-dotnet/rspec/vbnet/S2367_vb.net.json [hunk 1]
```diff
@@ -10,5 +10,6 @@
     "convention"
   ],
   "defaultSeverity": "Minor",
-  "ruleSpecification": "RSPEC-2367"
+  "ruleSpecification": "RSPEC-2367",
+  "sqKey": "PublicConstantName"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S2367_vb.net.json]: that's weird

File: sonaranalyzer-dotnet/rspec/vbnet/S2369_vb.net.json [hunk 2]
```diff
@@ -10,5 +10,6 @@
     "convention"
   ],
   "defaultSeverity": "Minor",
-  "ruleSpecification": "RSPEC-2369"
+  "ruleSpecification": "RSPEC-2369",
+  "sqKey": "PublicFieldName"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S2369_vb.net.json]: that's weird

File: sonaranalyzer-dotnet/rspec/vbnet/S2370_vb.net.json [hunk 3]
```diff
@@ -10,5 +10,6 @@
     "convention"
   ],
   "defaultSeverity": "Minor",
-  "ruleSpecification": "RSPEC-2370"
+  "ruleSpecification": "RSPEC-2370",
+  "sqKey": "PublicReadOnlySharedFieldName"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S2370_vb.net.json]: that's weird

File: sonaranalyzer-dotnet/rspec/vbnet/S2373_vb.net.json [hunk 4]
```diff
@@ -10,5 +10,6 @@
     "convention"
   ],
   "defaultSeverity": "Minor",
-  "ruleSpecification": "RSPEC-2373"
+  "ruleSpecification": "RSPEC-2373",
+  "sqKey": "TypeParameterName"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S2373_vb.net.json]: That's weird

File: sonaranalyzer-dotnet/rspec/vbnet/S2374_vb.net.json [hunk 5]
```diff
@@ -10,5 +10,6 @@
     "pitfall"
   ],
   "defaultSeverity": "Critical",
-  "ruleSpecification": "RSPEC-2374"
+  "ruleSpecification": "RSPEC-2374",
+  "sqKey": "UnsignedUsage"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S2374_vb.net.json]: That's weird

File: sonaranalyzer-dotnet/rspec/vbnet/S2375_vb.net.json [hunk 6]
```diff
@@ -10,5 +10,6 @@
     "clumsy"
   ],
   "defaultSeverity": "Minor",
-  "ruleSpecification": "RSPEC-2375"
+  "ruleSpecification": "RSPEC-2375",
+  "sqKey": "WithUsageCandidate"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S2375_vb.net.json]: That's weird

File: sonaranalyzer-dotnet/rspec/vbnet/S3385_vb.net.json [hunk 7]
```diff
@@ -11,5 +11,6 @@
     "bad-practice"
   ],
   "defaultSeverity": "Major",
-  "ruleSpecification": "RSPEC-3385"
+  "ruleSpecification": "RSPEC-3385",
+  "sqKey": "ExitStatementUsage"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S3385_vb.net.json]: That's weird

File: sonaranalyzer-dotnet/rspec/vbnet/S2429_vb.net.json [hunk 8]
```diff
@@ -10,5 +10,6 @@
     "clumsy"
   ],
   "defaultSeverity": "Minor",
-  "ruleSpecification": "RSPEC-2429"
+  "ruleSpecification": "RSPEC-2429",
+  "sqKey": "ImmediateArrayInitialization"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S2429_vb.net.json]: That's weird

File: sonaranalyzer-dotnet/rspec/vbnet/S2951_vb.net.json [hunk 9]
```diff
@@ -11,5 +11,6 @@
     "clumsy"
   ],
   "defaultSeverity": "Minor",
-  "ruleSpecification": "RSPEC-2951"
+  "ruleSpecification": "RSPEC-2951",
+  "sqKey": "ExitSelectRedundantUsage"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S2951_vb.net.json]: That's weird

File: sonaranalyzer-dotnet/rspec/vbnet/S2366_vb.net.json [hunk 10]
```diff
@@ -10,5 +10,6 @@
     "convention"
   ],
   "defaultSeverity": "Minor",
-  "ruleSpecification": "RSPEC-2366"
+  "ruleSpecification": "RSPEC-2366",
+  "sqKey": "PropertyName"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S2366_vb.net.json]: That's weird

File: sonaranalyzer-dotnet/rspec/vbnet/S2364_vb.net.json [hunk 11]
```diff
@@ -10,5 +10,6 @@
     "convention"
   ],
   "defaultSeverity": "Minor",
-  "ruleSpecification": "RSPEC-2364"
+  "ruleSpecification": "RSPEC-2364",
+  "sqKey": "PrivateFieldName"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S2364_vb.net.json]: That's weird

File: sonaranalyzer-dotnet/rspec/vbnet/S2363_vb.net.json [hunk 12]
```diff
@@ -10,5 +10,6 @@
     "convention"
   ],
   "defaultSeverity": "Minor",
-  "ruleSpecification": "RSPEC-2363"
+  "ruleSpecification": "RSPEC-2363",
+  "sqKey": "PrivateReadOnlySharedFieldName"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S2363_vb.net.json]: That's weird

File: sonaranalyzer-dotnet/rspec/vbnet/S2362_vb.net.json [hunk 13]
```diff
@@ -10,5 +10,6 @@
     "convention"
   ],
   "defaultSeverity": "Minor",
-  "ruleSpecification": "RSPEC-2362"
+  "ruleSpecification": "RSPEC-2362",
+  "sqKey": "PrivateConstantName"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S2362_vb.net.json]: That's weird

File: sonaranalyzer-dotnet/rspec/vbnet/S2359_vb.net.json [hunk 14]
```diff
@@ -10,5 +10,6 @@
     "bad-practice"
   ],
   "defaultSeverity": "Major",
-  "ruleSpecification": "RSPEC-2359"
+  "ruleSpecification": "RSPEC-2359",
+  "sqKey": "OnErrorStatement"
```
Comment (valhristov) [inline sonaranalyzer-dotnet/rspec/vbnet/S2359_vb.net.json]: That's weird

**Comment by Evangelink** [comment]
I have removed all legacy keys for vbnet rules on RSPEC (see discussion on [build topic](https://groups.google.com/a/sonarsource.com/d/topic/build/kGfqZnGCJ_w/discussion))

**Pull request merged**